demos/object_detection_demo/jupyter-python/detection_utils.py

```diff
@@ -0,0 +1,88 @@
+import random
```

Comment:
The name of the directory should be `jupyter_python` (with an underscore). That would be consistent with `cpp_gapi`, which we have elsewhere.

Reply:
Changed to `jupyter_python`